packages/0018-package-format.md

```diff
@@ -0,0 +1,150 @@
+# Project UmbGet Package format 📦
+
+Request for Contribution (RFC) 18 : _Package Format_
+
+## Code of conduct
+
+Please read and respect the [RFC Code of Conduct](https://github.com/umbraco/rfcs/blob/master/CODE_OF_CONDUCT.md)
+
+## Intended Audience
+
+The intended audience for this RFC is: Package developers and consumers
+
+## Summary
+
+Umbraco currently has two used package formats: 
+- The out of the box Umbraco ZIP package that can be generated from the backoffice
+- The NuGet package that is the standard in .NET
+
+This RFC's purpose is to suggest a way to bridge the gap and unify the two package formats while also looking closer at what a modern package format *should* offer.
+
+## Motivation
+
+A lot of package developers end up publishing their package as both a Zip and a NuGet package as both formats have unique advantages. However, as a developer, working with NuGet packages are highly preferred, as it has some very essential features - the main one being automatic dependency handling.
+
+We want to make it easier and simpler to create packages by only maintaining one format while getting the best features of both.
+
+## Detailed Design
+
+The high-level overview is that we want to remove the zip package format and move to a NuGet zip (.nupkg), and then make sure that that the benefits of the zip format is maintained:
+
+- Listing in the backoffice
+- Installing from the backoffice
+- Ability to install Umbraco content and schema
+- Ability to run code within the Umbraco context on install and uninstall
+
+We have a more detailed list of features we want the new package format to support below. If you feel something is missing please check the **Out of scope** section before commenting, we may have the same idea but want to leave it out for this first iteration!
+
+### Creating a package
+
+#### Umbraco content & media
+
+We want to ensure that we don't take a step back with functionality compared to what you can do today. One of the things you can do in an Umbraco Zip package but can't do in a NuGet package today is to include content and schema (and media with a lot of extra work). 
+
+We want to ensure that when you create a package in the backoffice you can include content and media (see schema below), to do that we will need to support serializing and deserializing content and media in the CMS.
+
+#### Umbraco schema
+
+Similar to the Umbraco content and media above, we want to keep the option of installing schema (document types, data types, templates, languages, etc.). 
```

Comment:
I think we should have a look at Deploy ? This is already proven technology on Umbraco cloud that exactly does this

Reply:
That could be an option, but isn't the package xml already doing this? What would the difference be? 🤔

Reply:
Yes the package xml already does that. But I think from a maintenance point of view one format would be better